spec-main/index.js

```diff
@@ -4,6 +4,13 @@ const v8 = require('v8');
 
 Module.globalPaths.push(path.resolve(__dirname, '../spec/node_modules'));
 
+// Extra module paths which can be used to load Mocha reporters
+if (process.env.MOCHA_MODULE_PATHS) {
```

Comment:
I wouldn't name this `MOCHA` as there's nothing really mocha-specific about this functionality. `TEST_EXTRA_MODULE_PATHS` maybe?

Reply:
Sure. Was just sticking with the naming convention of the other environment variables, but you're right, it's more generic.